main.py

```diff
@@ -79,7 +79,30 @@ async def on_message(message: Message):
             await message.channel.send("You need to use !list first, sugar.")
             return
 
-        await command_play(message)
```

Comment:
Oh, and could you update the README to include the new syntax?

Reply:
Done :)